### PR TITLE
initials

### PR DESCRIPTION
**Pull request by atmilich**

### CODE REVIEW

**Comment by atmilich** [comment]
initials & questions assignment 1

File: assignment1.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+1. The 'return 0' at the end of the code tells the compiler that the code is done. Since main is initialized as an int, it needs the 0 to tell the compiler it can stop compiling.
```
Comment (oldclesleycode) [inline assignment1.txt:1]: Not only does it tell the compiler that it's done, but that it has 0 errors - the rest of your answer is right!

File: assignment1.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+2. The code doesn't have '#include <stdio.h>' so it does not import the standard input/output library. Also, since x is declared as a float, the print line should use %f instead of %d.
```
Comment (oldclesleycode) [inline assignment1.txt:3]: Correct! You're missing one last error - see if you can find it.

File: assignment1.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+3. The equation as written incorrectly uses the order of operations. Instead, the equation should use parentheses after the -G so that it reads g = -G (m/r^2).
```
Comment (oldclesleycode) [inline assignment1.txt:5]: This is true, but you're missing a couple of errors! See if you can find them.

File: assignment1.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+4. ++var and var++ are very similar except the first uses prefix notation while the other uses postfix so the first returns the value after incrementing while the second returns the value before incrementing (so the value of var is different).
```
Comment (oldclesleycode) [inline assignment1.txt:7]: Correct! the order of operation is what changes!

File: initials.c [hunk 5]
```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
```
Comment (oldclesleycode) [inline initials.c:1]: Great job! Your code compiled and ran - the only thing I'd add is a return 0; and a comment at the top of your code with your name and function of your program.

**Pull request closed without merging**